ddtrace/bootstrap/sitecustomize.py

```diff
@@ -142,6 +54,21 @@ def cleanup_loaded_modules(aggressive=False):
 
 log = get_logger(__name__)
 
+if os.environ.get("DD_GEVENT_PATCH_ALL") is not None:
+    deprecate(
+        "The environment variable DD_GEVENT_PATCH_ALL is deprecated and will be removed in a future version. ",
+        postfix="There is no special configuration necessary to make ddtrace work with gevent.",
+        removal_version="2.0.0",
+    )
+if gevent.monkey.is_module_patched("sys"):
+    warnings.warn(
+        "Loading ddtrace after gevent.monkey.patch_all() is not supported and is "
+        "likely to break the application. Use ddtrace-run to fix this, or "
+        "import ddtrace.auto before calling gevent.monkey.patch_all().",
+        RuntimeWarning,
+    )
```

Comment:
we probably don't want to import gevent in here right? could it have side effects for an env which has gevent installed, but not used?

we could do something like this

```python
# Check if gevent has been imported already
if "gevent" in sys.modules:
    import gevent.monkey
    if gevent.monkey.is_module_patched("sys"):
        warnings.warn(...)
```

Aside, why check "sys" module? "threading" seems to be the more important one, but is `sys` always patched or something where it is more consistent to check?